earthly/cspell/Readme.md

```diff
@@ -0,0 +1,78 @@
+# CSpell Linter
+
+This Earthly Target and UDC enables uniform spell checking of all source and
+documentation files.
+
+## How it works
+
+Spellchecking is performed with the [`cspell`](cspell.org) program.
+
+Use the `CSPELL` Earthly UDC to enable uniform and consistent spell checking.
+
+This spellchecker is to be used in preference to tool specific spell checkers,
+such as `cargo spellcheck`.
+This is because we need to provide uniform and consistent spell checking across
+multiple source languages and documentation files.
+Tool specific spell checkers are limited in the kinds of files they can check,
+and also will use different configurations, dictionaries and project word lists.
+
+## Invocation
+
+In an Earthfile in your repo, add the following:
+
+```earthfile
+spellcheck-lint:
+    # Check spelling in this repo.
+    LOCALLY
+
+    DO github.com/input-output-hk/catalyst-ci:v1.2.0/earthly/cspell+CSPELL --src=$(echo ${PWD})
+```
+
+In this use case, the UDC is run Locally, so that the src in the repo can be directly checked.
+
+## Configuration
+
+Each repo will need a [`cspell.json`](http://cspell.org/configuration/) file in the root of the repo.
```

Comment:
Github offers "template repositories" which might make this process a bit smoother. Or we can use something like cookie cutter to generate template repos.

Reply:
We will have 5 Repos total this applies to.
We need to be really conscious about spending time to automate things that are trivial in practice and happen once.
I am sure I can copy this config file into those 5 repos faster than we can automate it.